integration-test/env/Extension/bin/_placeholder

```diff
@@ -0,0 +1 @@
+placeholder for git to preserve this directory
```

Comment:
@paulmey but the directory does not appear in .git at all, for documentation purposes (see README.md) I thought this dir better be present when the repo is checked out?